linera-views/src/backends/value_splitting.rs

```diff
@@ -454,22 +475,20 @@ impl WritableKeyValueStore for LimitedTestMemoryStore {
         if !batch.check_value_size(Self::MAX_VALUE_SIZE) {
             panic!("The batch size is not adequate for this test");
         }
-        self.store.write_batch(batch).await
+        self.inner.write_batch(batch).await
     }
 
     async fn clear_journal(&self) -> Result<(), MemoryStoreError> {
-        self.store.clear_journal().await
+        self.inner.clear_journal().await
     }
 }
 
 #[cfg(with_testing)]
 impl LimitedTestMemoryStore {
     /// Creates a `LimitedTestMemoryStore`
     pub fn new() -> Self {
-        let namespace = generate_test_namespace();
-        let store =
-            MemoryStore::new_for_testing(TEST_MEMORY_MAX_STREAM_QUERIES, &namespace).unwrap();
-        LimitedTestMemoryStore { store }
+        let inner = MemoryStore::new_for_testing();
+        LimitedTestMemoryStore { inner }
```

Comment:
What is a little puzzling here is that the maximal size is not showing up.
The tests show that the splitting is still working correctly. But then is the splitting till occurring for the `MemoryStore`.

Reply:
That's `WritableKeyValueStore::MAX_VALUE_SIZE` (not changed in the PR)